client/src/pages/Home.js

```diff
@@ -1,6 +1,8 @@
 // @flow
```

Comment:
i'm not sure how flow works but I think we should use either flow or prop-types for type-checking, both seems a little overkill